src/screens/Signup/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import SignupForm from '../../components/SignupForm';
+
+import styles from './Signup.module.scss';
+
+function Signup() {
+  return (
+    <div className={styles.signup}>
```

Comment:
this div and file is not necessary, put this styles on `SignupForm` div container